agent/remote_exec_test.go

```diff
@@ -107,6 +107,15 @@ func TestRemoteExecGetSpec_ACLToken(t *testing.T) {
 	testRemoteExecGetSpec(t, cfg)
 }
 
+func TestRemoteExecGetSpec_ACLAgentToken(t *testing.T) {
+	t.Parallel()
+	cfg := TestConfig()
+	cfg.ACLDatacenter = "dc1"
+	cfg.ACLAgentToken = "root"
+	cfg.ACLDefaultPolicy = "deny"
+	testRemoteExecGetSpec(t, cfg)
+}
+
 func testRemoteExecGetSpec(t *testing.T, c *Config) {
 	a := NewTestAgent(t.Name(), nil)
```

Comment:
Oh good catch - yeah this must be working because ACLs aren't getting enabled for either case. This is supposed to make sure we can get the token from the ACL agent token, and the other test checks the ACL token. I'll pass that through and make sure it looks like it's properly using ACLs.